src/misc/setup.cpp

```diff
@@ -1029,10 +1035,17 @@ parse_environ_result_t parse_environ(const char * const * envp) noexcept
 	return props_to_set;
 }
 
-void Config::ParseEnv(char **envp)
+void Config::ParseEnv()
 {
-	const auto env_filtered = parse_environ(envp);
-	for (const auto &set_prop_desc : env_filtered) {
+#ifdef _MSC_VER
+	char **envp = _environ;
+#else
+	char **envp = environ;
```

Comment:
Is it possible to use `const`'s of pointers to pointers?

Reply:
Purpose of this env variable is to give a common name to `*environ`, which is `char**`. But `parse_environ` function already treats this as if pointers were const :) In fact it was quite difficult to come up with correct combination of `const` keywords that work in GCC, Clang, and MSVC - compilers were very much trying to convince me to drop consts in that function :(

Reply:
Hmm, I'll try decorating `envp` with consts anyway.